.zappr.yaml

```diff
@@ -0,0 +1,15 @@
+approvals:
+  minimum: 1
+  ignore: pr_opener
+  pattern: "^([aA][cC][kK]|[lL][gG][tT][mM]|👌|:ok_hand:|👍|:\\+1:|:thumbsup:|:shipit:)$"
```

Comment:
I think there are two many. Somebody can accidentally approve. 
I think we only need `:shipit:` and ":+1:"
(and `:-1:` for disapproval)